event/listener.php

```diff
@@ -142,6 +142,7 @@ public function pwa_manifest()
 		$this->template->assign_vars([
 			'U_MANIFEST_URL'	=> $this->controller_helper->route('phpbb_webpushnotifications_manifest_controller'),
 			'U_TOUCH_ICON'		=> $this->config['pwa_icon_small'],
+			'SHORT_SITE_NAME'	=> $this->config['pwa_short_name'] ?: substr($this->config['sitename'], 0, 12),
```

Comment:
I'd then also make an addition to `PWA_SHORT_NAME_EXPLAIN` lang entry to reflect something like "If not set, 1st 12 characters of the 'Site name' setting above will be used`.